docs/visualize/tsvb.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[TSVB]]
+== Visualizing your data with TSVB
+
+TSVB is a time series data visualizer that allows you to use the full power of the 
+Elasticsearch aggregation framework. With TSVB, combine an infinite 
+number of aggregations and pipeline aggregations to display complex data in a 
+meaningful way.
+
+NOTE: In Elasticsearch version 7.3.0 and later, the time series data visualizer is now referred to simply as TSVB instead of Time Series Visual Builder.
+
+TSVB comes with these types of visualizations:
```

Comment:
Since this is a long list, how about adding a head 2, maybe "Visualization types", and using h3 for the types.  And maybe this for the intro:  

TSVB comes with six types of visualizations

Reply:
I had it your way before, but I like this way better :)